containers/runtime/README.md

```diff
@@ -0,0 +1,53 @@
+# Runtime Container Images
+
+## [cxx](cxx/)
+
+Base Image: [debian:stretch](https://hub.docker.com/_/debian)
+
+Working Directory: `/src/workspace`
+
+Entrypoint: `bash`
+
+Official Docker Image for the Debian distribution, decorated with autoconf,
+build-essential, clang, git, make, libtool, libgflags-dev and pkg-config.
+
+## [driver](driver/)
+
+Base Image: [debian:stretch](https://hub.docker.com/_/debian)
+
+Working Directory: `/src/workspace`
+
+Entrypoint: `bash`
+
+Official Docker Image for the Debian distribution, decorated with autoconf,
```

Comment:
you don't need to list what's in the docker image - everyone can look at the dockerfile.  Listing it here only leads to duplicate information (which is going to be pain to maintain and is almost guaranteed to get out of sync).

Having a README.md is useful if there is some general info, but don't mention what's obvious from the Dockerfiles themselves. You can mention metainformation like "C++ docker image is based off of the lastest debian stable"

You can also link to the interop base images from grpc/grpc repostitory for comparison (because what you have is basically an alternative to them).